engines/python/setup/setup.py

```diff
@@ -44,7 +44,7 @@ def run(self):
 
     requirements = ['psutil', 'packaging', 'wheel']
 
-    test_requirements = ['numpy', 'requests', 'Pillow']
+    test_requirements = ['numpy', 'requests', 'Pillow', 'transformers', 'torch']
```

Comment:
I don't think Python engine should depends on torch or transformers. DJL Python engine should be generic.

One option could be create another pip package and install only in DLC.

Reply:
But these are for test requirements, for running unit test cases.